ch(travis): Integrate Travis CI to work with Github

XROLE opened the pull request:
#### What does this pull request do?
- This connects Authors-Haven with Travis CI and instructs Travis to run all test written.
#### Description of task to be completed?
- [x] connect Travis CI with Github
- [x] add .travis.yml file
- [x] configure  .travis.yml file to run node js
- [x] configure .travis.yml file to run test
#### How should this be manually tested?
> - Clone the repository
> - Create a new off the develop branch
> - Make changes to the new branch, commit and push to remote
> - Create a pull request. 
- On successful creating a pull request, you will see a spinning yellow circle on pippin-ah-backend GitHub page telling you that Travis is running some background check
 #### What are the relevant pivotal tracker stories?
- [Finishes #162727540]

Commit:
ch(travis): Integrate Travis CI to work with Github
connect travis CI with Github
add .travis.yml file
configure  .travis.yml file to run node js
configure .travis.yml file to run test
[Finishes #162727540]

oreoluwade commented on .travis.yml:
```diff
@@ -0,0 +1,7 @@
+before_script:
+  - npm install
+script: 
+  - npm run test
```
Travis should still work as expected even if there's no script directive in this `.travis.yml`, so you could take that out.

By default, it is designed to find and run `npm test`

XROLE replied:
Ok, I never knew about that. Thanks for the feedback. I will be taken it out asap.

oreoluwade replied:
Oh, that's fine @XROLE I know you wouldn't have put it there if you knew.

XROLE replied:
#### Please, I need clarification on this;
- The build passed on Travis CI successfully but the badge has not changed color to show that the test is passing. Do I still add the badge that way or do I wait untill the badge color changes to success (green)?

XROLE replied:
> I'd need you to add the travis badge to the `README.md` @XROLE

Ok. I am adding it right away.

oreoluwade replied:
> #### Please, I need clarification on this;
> * The build passed on Travis CI successfully but the badge has not changed color to show that the test is passing. Do I still add the badge that way or do I wait until the badge color changes to success (green)?

The reason the badge is not changing color is that we need the develop version of the CI. However, at the moment, travis is not on develop yet, so you'd most likely only see an `unknown` status. As long as the travis build passes, when we merge to develop, the badge will be green.

XROLE replied:
Ok. Thanks so much for that.

oreoluwade commented:
I'd need you to add the travis badge to the `README.md` @XROLE

Commit:
CH(Travis): Removed unnecesory script command
removed npm install command from before_script hook in .travis.yml file
remove npm run test command from the script command in .travis.yml file
[Fixes #162727540]

Commit:
CH(Travis): add travis badge
add Travis CI badge to the project readme file
[Fixes #162727540]

XROLE commented:
> I'd need you to add the travis badge to the `README.md` @XROLE

Readme badge added successfully

oreoluwade commented on README.md:
```diff
@@ -1,4 +1,5 @@
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
+[![Build Status](https://travis-ci.org/andela/pippin-ah-backend.svg?branch=develop)](https://travis-ci.org/andela/pippin-ah-backend)
```
Please, swap the position of both badges, then when you do that for coveralls as well, put it before the hound badge.

XROLE replied:
Ok. I'll do it right away

XROLE replied:
Travis CI badge position swapped successfully.

Commit:
CH(Travis): Swapped position of Travis badge
Moved Travi CI badge to the first position
Moved Hound badge to the second position
[Fixes #162727540]
